src/components/NewGameButtons.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import React from "react";
+
+export default function NewGameButtons() {
```

Comment:
We don't need this abstraction:

It should be something like this in `Home.tsx` component:

```
<main className="flex ...">
        <div className="w-full flex flex-col gap-10">
          <Image
            src={logo}
            alt="x"
            width="80"
            height="80"
            className="self-center"
          />
          <ChooseMarkSwitch />
          <div className="flex flex-col gap-6">
            <Button1WithAGoodName />
            <Button2WithAGoodName />
          </div
        </div>
      </main>



```